config/crds/azureprovider_v1alpha1_azureclusterproviderstatus.yaml

```diff
@@ -0,0 +1,181 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: azureclusterproviderstatuses.azureprovider.k8s.io
+spec:
+  group: azureprovider.k8s.io
+  names:
+    kind: AzureClusterProviderStatus
+    plural: azureclusterproviderstatuses
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          type: string
+        bastion:
+          properties:
+            ebsOptimized:
```

Comment:
AWS specific I assume?

Reply:
Yep, I'll be cleaning up the types in a separate PR!